README.md

```diff
@@ -4,6 +4,7 @@
 
 > Portfolio made by Ntare Guy
 
+
 ## Visit the page [here](https://41hulk.github.io/micro-portfolio/)
```

Comment:
The website is deployed using GitHub pages 🔥😎 Working is expected!

Reply:
Thanks, @mateo951 🚀